summit-2019-2.html

```diff
@@ -110,6 +110,17 @@ <h1 class="section-heading">Register here</h1>
         </p>
       </div>
     </section>
-    <div data-include="../includes/footer.html"></div>
+    <section>
```

Comment:
I guess there isn't any need for any "Past summits" section since this is the oldest one we've archived?

Reply:
My thought was that "past" was relative to today and not the summit in question, i.e. the list would contain all past summits to make them all more accessible.

Reply:
Or maybe "Other summits"? Then it makes sense to link to the current summit (i.e. summit.html) which may or may not have appeared in the past.

Listing the summits in a single file that's included from all files is certainly an option, but then all summit pages will have a link to itself. Given the low frequency of summits I figured we could manage to keep that list in sync by hand.

Reply:
Agreed. "Other summits" sounds good to me.

Reply:
Done. Reviewing commit stacks on GH is a travesty as usual but I merged from the parent branch into this one and added a couple of commits on top.